### DISCUSSION TITLE
MoeMon support

### DISCUSSION THREAD

**Discussion by lxlGhostlylxl** [besteon/Ironmon-Tracker#491] [Ideas]
I'm not really sure if this is possible, but it would be really nice if this tracker would work with the moemon versions of the game.

**Comment by UTDZac** [comment]
Hi,

In what way are you wanting the Tracker to work with these games?

I did [a little digging](https://www.reddit.com/r/moemon/comments/1b3f4t1/all_you_need_to_know_about_moemon/) and I found the sprite replacement patch for MoeMon, applied it to my FireRed ROM, and tested it out on the Tracker. It appears to work fine... other than the obvious that the Meomon sprites don't render on the Tracker screen itself (we only have original pokemon sprites).

![image](https://github.com/user-attachments/assets/43c0537d-d9b0-492d-a0da-d1cfdf2f3f99)
